bugsnag/bugsnag-js
# [PLAT-4935] Add automate-symbolication command (Android)

Add a command to automate source map upload when a React Native build happens.

Unit tests included for relevant parts. Interactive CLI aspects will be test via maze runner at a later point.

To test this out, use the following steps:

- `git fetch && git checkout feature/rn-cli-automate-symbolication-cmd`
- `npm i`
- `npm run bootstrap`
- `npm run build`
- `cd packages/react-native-cli`
- `npm link` (this symlinks the local package to your global npm modules)

On a React Native project:

- check that the command is linked up correctly `bugsnag-react-native-cli --help` 
- run `bugsnag-react-native-cli automate-symbolication`
